### ISSUE TITLE
Form Wizards via API

### ISSUE THREAD

**Issue by demslob** [barseghyanartur/django-fobi#205]
Hello! Do you have any plans on implementing wizards in DRF API?
Thank you.

**Comment by barseghyanartur** [comment]
@demslob:

Nope, but I could do that if sponsored. Contact me for details (see the bottom of the readme).

**Comment by barseghyanartur** [comment]
@demslob:

I'm leaving this open (as possible improvement). Note, that current stable branch is `v0.13.x`. Master contains work-in-progress migration to class-based-views.
